test/helpers/test_qiskit_to_ionq.py

```diff
@@ -190,3 +199,134 @@ def test_full_circuit(simulator_backend):
     assert actual_metadata_header == expected_metadata_header
     assert actual_output_map == expected_output_map
     assert actual == expected_rest_of_payload
+
+
+def test_circuit_transpile(simulator_backend):
```

Comment:
love to assert the error cases. brings joy to my cold heart.

Reply:
:100: